### DISCUSSION TITLE
How to install libgranite-7-dev

### DISCUSSION THREAD

**Discussion by joeljosedev** [elementary/granite#576] [Q&A] [answered]
I want to compile [elementary/shortcut-overlay](https://github.com/elementary/shortcut-overlay) from source, but `libgranite-7-dev` is a dependency and I cannot find a way to download it. I did try using libgranite-common and libgranite-dev but while compiling, meson reported the following error:

```
Run-time dependency granite-7 found: NO (tried pkgconfig)

meson.build:26:0: ERROR: Dependency "granite-7" not found, tried pkgconfig
```

**Comment by danirabbit** [comment, marked as answer]
Granite 7 is currently only available in the daily development snapshots in early access or by manually downloading and compiling its source code.

It will be available as part of our Flatpak runtime version 7 and in OS 7 when those are released
